lib/pdf_fill/extras_generator_v2.rb

```diff
@@ -75,6 +81,46 @@ def sorted_subquestions_markup
       end
     end
 
+    class CheckedDescriptionQuestion < Question
```

Comment:
This class also needs some more testing. 
![image](https://github.com/user-attachments/assets/2a7bd3c8-731c-459d-b499-b9fde7c647e8)